Update 02-setup.md (#123)
MetOffice/git-novice: episodes/02-setup.md

It is possible to reconfigure the text editor for Git whenever you want to change it.

::: caution

### Nedit not available on Azure Spice

Nedit won't be available on Azure SPICE VDI.
Details are available in the [Met Office Azure SPICE Documentation](https://wwwspice/docs/software/removed/?h=nedit).

> Your closest editor in the table above is Gedit.

Author: ```suggestion
The most similar editor in the table above is Gedit.
```
